omod/src/main/webapp/master-single-page-application.jsp

```diff
@@ -5,6 +5,8 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>OpenMRS</title>
+    <link rel="icon" type="image/ico" href="/openmrs/images/openmrs-favicon.ico"/>
```

Comment:
The more common way is to use `rel="shortcut icon"` and `type="image/x-icon"` (see [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML) and [here](https://stackoverflow.com/questions/6644684/do-you-have-to-include-link-rel-icon-href-favicon-ico-type-image-x-icon)). I'm not sure what differences, if any, there would be using what you have here. But would prefer just to do the more common pattern for favicons.

```suggestion
    <link rel="shortcut icon" type="image/x-icon" href="/openmrs/images/openmrs-favicon.ico"/>
```